examples/computer_vision/byol_pytorch/lars.py

```diff
@@ -0,0 +1,174 @@
+"""
+Layer-wise adaptive rate scaling for SGD in PyTorch.
+Adapted from https://github.com/untitled-ai/self_supervised/blob/master/lars.py
```

Comment:
same here.  if this file is the same as the source, i prefer we git clone and checkout a specific commit into the working directory as part of the startup-hook so we don't have to maintain this portion.

Reply:
Done.